### PR TITLE
Fix row named format for binary inputs in REST API & fix logging for MediaPipe stream info

### PR DESCRIPTION
**Pull request by mzegla**

### CODE REVIEW

File: src/mediapipe_internal/mediapipe_utils.hpp [hunk 1]
```diff
@@ -29,6 +29,9 @@ extern const std::string OV_TENSOR_PREFIX;
 extern const std::string OVMS_PY_TENSOR_PREFIX;
 extern const std::string MP_IMAGE_PREFIX;
 
-std::pair<std::string, mediapipe_packet_type_enum> getStreamNamePair(const std::string& streamFullName);
+enum MediaPipeStreamType { INPUT,
```
Comment (dkalinowski) [inline src/mediapipe_internal/mediapipe_utils.hpp:32]: Prefer enum class over C enums

File: src/mediapipe_internal/mediapipe_utils.cpp [hunk 2]
```diff
@@ -40,7 +40,17 @@ const std::string MP_IMAGE_PREFIX{"IMAGE"};
 
 const std::string EMPTY_STREAM_NAME{""};
 
-std::pair<std::string, mediapipe_packet_type_enum> getStreamNamePair(const std::string& streamFullName) {
+std::pair<std::string, mediapipe_packet_type_enum> getStreamNamePair(const std::string& streamFullName, const MediaPipeStreamType& streamType) {
```
Comment (dkalinowski) [inline src/mediapipe_internal/mediapipe_utils.cpp:43]:
```suggestion
std::pair<std::string, mediapipe_packet_type_enum> getStreamNamePair(const std::string& streamFullName, MediaPipeStreamType streamType) {
```
No need to pass integers via const reference

File: src/mediapipe_internal/mediapipe_utils.cpp [hunk 3]
```diff
@@ -40,7 +40,17 @@ const std::string MP_IMAGE_PREFIX{"IMAGE"};
...
+    std::string streamTypeStr;
+    switch (streamType) {
+    case MediaPipeStreamType::INPUT:
```
Comment (dkalinowski) [inline src/mediapipe_internal/mediapipe_utils.cpp:46]: Move enum<->string mapping out of this method? Probably to mediapipe_utils?

File: src/test/tfs_rest_parser_row_test.cpp [hunk 4]
```diff
@@ -122,6 +122,16 @@ TEST(TFSRestParserRow, InvalidShape_1D) {
         {"i":[[12.0]]}, {"i":13.0}
     ]})"),
         StatusCode::REST_COULD_NOT_PARSE_INSTANCE);
+
+    EXPECT_EQ(parser.parse(R"({"signature_name":"","instances":[
```
Comment (dkalinowski) [inline src/test/tfs_rest_parser_row_test.cpp:126]: Missing positive cases which relate to the original bug cause

File: src/test/tfs_rest_parser_row_test.cpp [hunk 5]
```diff
@@ -109,6 +116,10 @@ TEST(TFSRestParserRow, ParseValid2Inputs) {
                                                               //============
                                                               11.0, 12, 13.0,
                                                               14.0, 15.0, 16.0));
+
+    char expectedBinary[] = {57, 28, 52};
+    EXPECT_EQ(inputC.string_val()[0], std::string(&expectedBinary[0], &expectedBinary[0] + 3));
```
Comment (dkalinowski) [inline src/test/tfs_rest_parser_row_test.cpp:121]:
```suggestion
    EXPECT_EQ(inputC.string_val()[0], std::string(expectedBinary, expectedBinary + 3));
```
Isnt it sufficient?

**Review by dkalinowski — APPROVED** [review]

File: src/test/tfs_rest_parser_row_test.cpp (hunk 5 above)
Comment (mzegla) [reply src/test/tfs_rest_parser_row_test.cpp:121]: That's correct, I'll change that in all places

**Review by michalkulakowski — APPROVED** [review]

**Pull request merged**